api/app.test.js

```diff
@@ -25,12 +25,12 @@ describe("POST /seedDatabase", () => {
   });
 });
 
-describe("POST /entry", () => {
+describe("POST /entries", () => {
   it("adds an item to the database", async () => {
     const startTime = new Date();
     const endTime = new Date();
     await supertest(app)
-      .post("/entry")
+      .post("/entries")
```

Comment:
is this adding or getting? is this one or two entries? I don't understand the test

Reply:
the .post shows that it's posting/adding, and the convention in RESTful routing is that the route is pluralized. More on that here: https://www.learnhowtoprogram.com/c-and-net/basic-web-applications/introduction-to-restful-routing